From 7f68a5edde0eab0d519aa510d77c3abdf35ee80d Mon Sep 17 00:00:00 2001
From: Kaushal Billore <kaushal.billore@intel.com>
Date: Wed, 26 Feb 2020 21:42:27 +0530
Subject: [PATCH] USB camera patches to support hotplug Feature

Signed-off-by: Kaushal Billore <kaushal.billore@intel.com>
---
 services/camera/libcameraservice/CameraService.cpp | 9 +--------
 1 file changed, 1 insertion(+), 8 deletions(-)

diff --git a/services/camera/libcameraservice/CameraService.cpp b/services/camera/libcameraservice/CameraService.cpp
index 2bf42b6..1afcf54 100644
--- a/services/camera/libcameraservice/CameraService.cpp
+++ b/services/camera/libcameraservice/CameraService.cpp
@@ -320,13 +320,6 @@ void CameraService::onDeviceStatusChanged(const String8& id,
         if (clientToDisconnect.get() != nullptr) {
             ALOGI("%s: Client for camera ID %s evicted due to device status change from HAL",
                     __FUNCTION__, id.string());
-            // Notify the client of disconnection
-            clientToDisconnect->notifyError(
-                    hardware::camera2::ICameraDeviceCallbacks::ERROR_CAMERA_DISCONNECTED,
-                    CaptureResultExtras{});
-            // Ensure not in binder RPC so client disconnect PID checks work correctly
-            LOG_ALWAYS_FATAL_IF(getCallingPid() != getpid(),
-                    "onDeviceStatusChanged must be called from the camera service process!");
             clientToDisconnect->disconnect();
         }
 
@@ -1716,7 +1709,7 @@ Status CameraService::supportsCameraApi(const String16& cameraId, int apiVersion
         case -1: {
             String8 msg = String8::format("Unknown camera ID %s", id.string());
             ALOGE("%s: %s", __FUNCTION__, msg.string());
-            return STATUS_ERROR(ERROR_ILLEGAL_ARGUMENT, msg.string());
+            break;
         }
         default: {
             String8 msg = String8::format("Unknown device version %x for device %s",
-- 
2.7.4

