From e45ca38d2dc78f94af3cbaabfe84c2ee5c0b8070 Mon Sep 17 00:00:00 2001
From: pmandri <padmasree.mandri@intel.com>
Date: Tue, 12 Nov 2019 18:43:23 +0530
Subject: [PATCH] Use TARGET_PRODUCT for cic audio hal

Tracked-On:
Signed-off-by: pmandri <padmasree.mandri@intel.com>
---
 primary/Android.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/primary/Android.mk b/primary/Android.mk
index a62f168..0c5cdfc 100644
--- a/primary/Android.mk
+++ b/primary/Android.mk
@@ -40,7 +40,7 @@ LOCAL_PROPRIETARY_MODULE := true
 
 LOCAL_HEADER_LIBRARIES += libhardware_headers
 
-LOCAL_MODULE := audio.primary.$(TARGET_BOARD_PLATFORM)
+LOCAL_MODULE := audio.primary.$(TARGET_PRODUCT)
 
 LOCAL_MODULE_RELATIVE_PATH := hw
 
-- 
2.17.1

